src/app/shared/pipes/short-url.pipe.ts

```diff
@@ -5,7 +5,7 @@ import { Pipe, PipeTransform } from '@angular/core';
 })
 export class ShortUrlPipe implements PipeTransform {
 
-  transform(value: string, trimLength: number = 200): string {
+  transform(value: string, trimLength: number = 160): string {
```

Comment:
Why is this changes to 160 from 200?

Reply:
as yesterday discussed the font was not visible so increased the size of the font and reduced the size of the pipe without breaking the ui